src/app/public/modules/ag-grid/cell-validator/ag-grid-cell-validator-tooltip.component.ts

```diff
@@ -0,0 +1,80 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, Input, NgZone } from '@angular/core';
+import { SkyPopoverMessage, SkyPopoverMessageType } from '@skyux/popovers';
+import { CellFocusedEvent, Events } from 'ag-grid-community';
+import { Subject } from 'rxjs';
+import { SkyCellRendererCurrencyParams } from '../types/cell-renderer-currency-params';
+
+@Component({
+  selector: 'sky-ag-grid-cell-validator-tooltip',
+  styleUrls: ['ag-grid-cell-validator-tooltip.component.scss'],
+  templateUrl: 'ag-grid-cell-validator-tooltip.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyAgGridCellValidatorTooltipComponent {
+  @Input()
+  public set params(value: SkyCellRendererCurrencyParams) {
```

Comment:
My comment regarding this being `SkyCellValidatorParams` is resolved; however, I'm still seeing this as `SkyCellRendererCurrencyParams`